### DISCUSSION TITLE
Clear errors on another product form when a different product form is submitted.

### DISCUSSION THREAD

**Discussion by james0r** [EvgeniyMukhamedjanov/liquid-ajax-cart#14] [Q&A] [answered]
Just migrated to version 1.10. You're doing a killer job, this thing is freaking awesome.

Got a question though, so as title says, on my collections page i want other product request errors to be hidden when any other product add-to-cart is attempted. What's the best way to handle this? via an attribute or listen for requests in JS? Thanks!

Store: https://store2-0-playground.myshopify.com/ 
Password: sprite

Code for each grid item --> https://gist.github.com/james0r/aa88aa856a82ecd151f375204ab98b4c

<img width="625" alt="image" src="https://user-images.githubusercontent.com/34731936/161407310-701fea8d-a7f4-4cc1-a4ba-d50f862d8893.png">

**Comment by EvgeniyMukhamedjanov** [comment, marked as answer]
Hey @james0r , thank you! I'm happy to hear that you continue to use Liquid Ajax Cart.
In order to remove the form error messages you can use the following snippet:
```javascript
subscribeToCartAjaxRequests( requestState  => {
  if ( requestState.requestType === 'add' ) {
    document.querySelectorAll('[data-ajax-cart-messages="form"]').forEach(element => element.innerHTML = '')
  }
}
```

I'll convert the issue to discussion.

**Reply by james0r** [reply]
You rule thank you! I realized later in the day that I posted it as an issue but was AFK. hah

**Reply by EvgeniyMukhamedjanov** [reply]
@james0r If you have released a store with the Liquid Ajax Cart — let me know, I'll add a link to the store to the liquid-ajax-cart.js.org website under the "Built with Liquid Ajax Cart" section :-)
